src/Eloquent/QueriesExpressions.php

```diff
@@ -9,7 +9,7 @@ trait QueriesExpressions
     /**
      * Get a new query builder instance for the connection.
      *
-     * @return \Illuminate\Database\Query\Builder
+     * @return Builder
```

Comment:
Please use fully qualified names:

```suggestion
     * @return \Staudenmeir\LaravelCte\Query\Builder
```